Mathlib/LinearAlgebra/AffineSpace/Independent.lean

```diff
@@ -1139,6 +1139,16 @@ lemma affineCombination_mem_closedInterior_iff {n : ℕ} {s : Simplex k P n} {w
     s.independent w' w hw' hw hww']
   exact hw'01
 
+lemma interior_subset_affineSpan {n : ℕ} {s : Simplex k P n} :
+    s.interior ⊆ affineSpan k (Set.range s.points) := by
+  rintro p ⟨w, hw, hi, rfl⟩
+  exact affineCombination_mem_affineSpan_of_nonempty hw _
+
+lemma closedInterior_subset_affineSpan {n : ℕ} {s : Simplex k P n} :
+    s.closedInterior ⊆ affineSpan k (Set.range s.points) := by
```

Comment:
Do we have that the interior is a subset of the closedInterior? If so, perhaps we only need this lemma and not the previous one.

I'll let you choose

bors d+

Reply:
Replaced with `interior_subset_closedInterior`.